Se desea tomar una muestra aleatoria estratificada de las personas mayores
de edad de un municipio, cuyos estratos son los siguientes intervalos de
edades, en años: de 18 a 30, de 31 a 45, de 46 a 60 y mayores de 60. En el
primer intervalo hay 7500 personas, en el segundo hay 8400, en el tercero
5700 y en el cuarto 3000.

Calcule el tamaño de la muestra total y su composición, sabiendo que el
muestreo se hace con afijación proporcional y se han elegido al azar 375
personas del primer estrato.

In [7]:
first_interval = 7500
second_interval = 8400
third_interval = 5700
fourth_interval = 3000

#Tamaño de la población
population_size = first_interval + second_interval + third_interval + fourth_interval

tamagno_sleccion_primer_estrato = 375

# Proporcion del primer estrato en relación a la población
proporcion_primer_estrato = first_interval  / population_size

#TTamaño total de la muestra
tamagno_total_muestra = tamagno_sleccion_primer_estrato / proporcion_primer_estrato

print(f"El tamaño de la muestra es: {tamagno_total_muestra} personas")

# Tamaño del segundo estrato

tamagno_sleccion_segundo_estrato = second_interval / population_size * tamagno_total_muestra
tamagno_sleccion_tercer_estrato = third_interval / population_size * tamagno_total_muestra
tamagno_sleccion_cuarto_estrato = fourth_interval / population_size * tamagno_total_muestra

# print Tamaño del primer estrato

print(f"Hay {tamagno_sleccion_primer_estrato} personas en el primer estrato")

# print Tamaño del segundo estrato

print(f"Hay {tamagno_sleccion_segundo_estrato: .0f} personas en el segundo estrato")

# print Tamaño del tercer estrato

print(f"Hay {tamagno_sleccion_tercer_estrato: .0f} personas en el tercer estrato")

# print Tamaño del cuarto estrato

print(f"Hay {tamagno_sleccion_cuarto_estrato: .0f} personas en el cuarto estrato")

El tamaño de la muestra es: 1230.0 personas
Hay 375 personas en el primer estrato
Hay  420 personas en el segundo estrato
Hay  285 personas en el tercer estrato
Hay  150 personas en el cuarto estrato


Si el contenido en gr. de un determinado medicamento X sigue una
distribución N(7.5,0.3), calcular la probabilidad de que para una muestra de
tamaño n=5, se obtenga medio menor que 7, Pr ( X ≤ 7).

In [10]:
from math import sqrt


sample_size = 5
mean = 7.5
standar_deviation = 0.3

z = (7-7.5)/(0.3/sqrt(5))

print(z)

-3.7267799624996503


si z = -3.7267799624996503
osea P(z < -3.7267799624996503) = P(z > 3.7267799624996503)
entonces buscamos en la tabla de z

1 - P(z > 3.7267799624996503) = 1 - 0.9999 = 0.0001

hay una probabilidad baja, de 0.01%, de que se obtenga un número menor a 7

-3.7267799624996503
